turtlebot2_drivers/launch/turtlebot_follow.py

```diff
@@ -31,3 +31,8 @@ def launch(launch_descriptor, argv):
         name='follower',
         exit_handler=restart_exit_handler,
     )
+    ld.add_process(
+        cmd=['joy_node'],
+        name='joy',
+        exit_handler=restart_exit_handler,
```

Comment:
@dhood identified a problem with the `restart_exit_handler` in #2. Can you try some other exit handlers to see what works well, then propagate the change to the other launch files?